File: ProjectEuler/Problem_1/sol.kt

var sum:Int = 0;
fun main(){

    for (i in 3..(1000-1)) {
        if (i%3 == 0){
            sum += i;
        } else if ( i%5 == 0) {
            sum += i;
        }
    }

    println(sum);
}

File: ProjectEuler/Problem_1/sol.ts

let sum:number = 0;

for (let i=3; i < 10; i++){
    if (i%3 == 0){
        sum += i;
        continue;
    } else if ( i%5 == 0) {
        sum += i;
    }

}

console.log(sum);

